Hestia/app/src/main/java/hestia/UI/ExpandableListAdapter.java

```diff
@@ -105,7 +105,12 @@ public boolean onMenuItemClick(MenuItem item) {
                                 //Settings
                                 break;
                             case R.id.delete:
-                                //c.deleteDevice((Device)((DeviceBar) getChild(groupPosition, childPosition)).getDevice())
+                                /**
```

Comment:
True, will add an issue to it. It would be nice to have a confirmation screen before actually deleting it.